doc/sphinx/language/core/sections.rst

```diff
@@ -131,6 +131,90 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    Notice the difference between the value of :g:`x'` and :g:`x''` inside section
    :g:`s1` and outside.
 
+.. _visibility-attributes-sections:
+
+Summary of locality attributes in a section
+-------------------------------------------
+
+The following table sums up the locality of vernacular commands in sections, when
+outside the section where it was entered.
+A similar table for :cmd:`Module` can be found
+:ref:`here <visibility-attributes-modules>`.
+
+.. list-table::
+  :header-rows: 1
+
+  * - ``Command``
+    - without attribute
+    - :attr:`local`
+    - :attr:`export`
+    - :attr:`global`
+
+  * - :cmd:`Definition`, :cmd:`Lemma`, :cmd:`Axiom`, ...
+    - same as :attr:`local`
+    - available
+    - attribute not supported
```

Comment:
```suggestion
    - available
    - not supported
```

Easier to read if the entries are more succinct.  Could you clarify the difference between "available" and "not supported" just before the table?  Does "not supported" mean you get an error message or that it's ignored?  (Ignored seems like it confuse the user.)  You could use just "✔" and "❌" to make it even easier to read.

Reply:
"not supported" means that there is an error message, yes. A ❌ could make it easier to read, you're right.